jws.go

```diff
@@ -75,7 +75,9 @@ type Signature struct {
 	original  *rawSignatureInfo
 }
 
-// ParseSigned parses a signed message in JWS Compact or JWS JSON Serialization.
+// ParseSigned parses a signed message in JWS Compact or JWS JSON Serialization. Validation fails if
+// the JWS is signed with an algorithm that isn't in the provided list of signature algorithms.
+// Applications should decide for themselves which signature algorithms are acceptable.
```

Comment:
"Should decide for themselves" is a bit "throw 'em in the ocean and they'l learn how to swim." :D

How about:

> If you're not sure which signature algorithms your application might receive, consult the documentation of the service that provides them or the protocol that you are implementing. You can also try getting an example JWS and decoding it with https://jwt.io/ to find the "alg" field.

Reply:
Good idea. I wordsmithed a bit to avoid suggesting that jwt.io is the singular tool you can use for this and to avoid using "service" (the JWT could have been generated by a client or some other thing we wouldn't think of as a service).